src/model/game.py

```diff
@@ -20,3 +20,6 @@ def __init__(
         self.players: list[Player] = players
         self.unit: str = unit
         self.is_open: bool = is_open
+
+    def game_is_tie(self) -> bool:
```

Comment:
This is a data class - it shouldn't contain any business rules such as tie condition.